api/graphql/schema.graphql

```diff
@@ -1383,6 +1439,14 @@ type Query {
     )
     @guard
     @canModel(ability: "viewBasicInfo", model: "User")
+  employeeWFAPaginatedAdminTable: [UserEmployeeWFA]!
+    @paginate(
+      defaultCount: 10
+      maxCount: 1000
+      scopes: ["whereAuthorizedToViewEmployeeWFAAdminTable"]
+      model: "App\\Models\\User"
+    )
+    @guard
```

Comment:
Sorry to keep poking around to find things, but I think the pagination query will need a `where` field to add filtering in possibly the next issue